astrocut/asdf_cutouts.py

```diff
@@ -41,8 +44,8 @@ def _get_cloud_http(s3_uri: Union[str, S3Path], key: str = None, secret: str = N
     url = f'https://{s3_path.bucket}.s3.amazonaws.com/{s3_path.key}'
     resp = requests.head(url, timeout=10)
     is_anon = False if resp.status_code == 403 else True
-    if verbose and not is_anon:
-        print(f'Attempting to access private S3 bucket: {s3_path.bucket}')
+    if not is_anon:
+        log.info(f'Attempting to access private S3 bucket: {s3_path.bucket}')
```

Comment:
instead of formatting the string and passing it into the logger, as you have here with the fstring, and below with the `str.format()` calls, it is best practice to let the logger do the formatting. this is for security (in some places we do call astrocut as a web service, so some of these inputs may be untrusted and we don't want them logged onto a server), but there's another reason as well. If the logging level is skipped (e.g. Debug) the logger can avoid doing any str formatting if it's passed in this way, whereas if you format the string before passing it into the logger, you are always doing that (small) bit of work.

so in this case, my suggestion would be:

```suggestion
        log.info('Attempting to access private S3 bucket: %s', s3_path.bucket')
```

And should be applied to all the log formatting here.